### PR TITLE
Signoff: Content feedback fixes

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**
Something weird happened with my #801 branch, same changes here.

### CODE REVIEW

**Review by edburyenegren-okta — APPROVED** [review]
lookin' good!

**Comment by edburyenegren-okta** [comment]
Re-reviewed - still good!

**Review by edburyenegren-okta — APPROVED** [review]
🚀

**Comment by edburyenegren-okta** [comment]
@arnoldsandoval-okta I just added a fix for https://github.com/okta/odyssey/issues/816 to this branch

File: packages/docs/.vuepress/public/images/anatomy-link.svg [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
...8.168 561.146 168.168C564.794 168.168 566.378 169.896 566.378 172.776V173.448C558.65 174.072 551.402 176.328 551.402 182.856C551.402 187.08 554.378 189.672 559.082 189.672ZM560.042 186.216C557.066 186.216 555.626 184.776 555.626 182.52C555.626 178.632 560.57 177.144 566.378 176.52V179.16C566.378 184.104 563.162 186.216 560.042 186.216ZM586.647 189H592.119V185.4H587.175C585.495 185.4 584.775 184.68 584.775 183V168.696H591.255V165.24H584.775V157.8H580.695V165.24H575.175V168.696H580.695V183.24C580.695 187.176 582.711 189 586.647 189ZM603.426 189.672C607.122 189.672 609.378 188.088 611.154 185.4C611.442 187.368 611.97 189 615.57 189H618.69V185.4H616.194C615.09 185.4 614.802 185.112 614.802 184.008V172.728C614.802 167.832 611.874 164.568 605.538 164.568C599.346 164.568 596.082 167.928 595.218 171.096L599.202 172.536C599.826 170.76 601.122 168.168 605.49 168.168C609.138 168.168 610.722 169.896 610.722 172.776V173.448C602.994 174.072 595.746 176.328 595.746 182.856C595.746 187.08 598.722 189.672 603.426 189.672ZM604.386 186.216C601.41 186.216 599.97 184.776 599.97 182.52C599.97 178.632 604.914 177.144 610.722 176.52V179.16C610.722 184.104 607.506 186.216 604.386 186.216Z" fill="#1662DD"/>
```
Comment (jmelberg-okta) [inline packages/docs/.vuepress/public/images/anatomy-link.svg:2]: What caused these changes? SVGO?

File: packages/vuepress-theme-odyssey/global-components/Description.vue [hunk 2]
```diff
@@ -15,12 +15,13 @@ export default {
   props: {
     variant: {
       type: String,
-      required: false
+      required: false,
+      default: null
```
Comment (jmelberg-okta) [inline packages/vuepress-theme-odyssey/global-components/Description.vue:19]: Explicitly adding this shouldn't be necessary. I'd assume `default` is `null` by default.

**Review by jmelberg-okta — APPROVED** [review]

File: packages/docs/.vuepress/public/images/anatomy-link.svg (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/docs/.vuepress/public/images/anatomy-link.svg:2]: These are content changes. Once this PR is merged, I'll double-check optimization in my SVG Opt PR.

File: packages/vuepress-theme-odyssey/global-components/Description.vue (hunk 2 above)
Comment (arnoldsandoval-okta) [reply packages/vuepress-theme-odyssey/global-components/Description.vue:19]: This one is weird to me as well, it is required by `eslint-config-vuepress`. It's something we should look to override in the future.

**Pull request merged**